docs/INSTALL.md

```diff
@@ -241,6 +241,11 @@ To check that `lnd` was installed properly run the following command:
 This command requires `bitcoind` (almost any version should do) to be available
 in the system's `$PATH` variable. Otherwise some of the tests will fail.
 
+**Tab completion for `lncli`**
```

Comment:
nit: Call this `Command-line completion` since the tabulator key is just the default key binding for it?